### DISCUSSION TITLE
HASS core not starting up on truenas

### DISCUSSION THREAD

**Discussion by QuentinIII** [tprelog/iocage-homeassistant#51] [Q&A]
Hi, after automatically install hass core onto truenas, the deamon won't start up. 

As I never work on BSD I need some help. When I go into the jail shell, all hass as to say is "homeassistant not running? (check /var/run/homeassistant_daemon.pid)"

Thanks for help !

**Comment by adorobis** [comment]
Can you check the log file (should be in `/var/log/homeassistant_daemon.log`)? See if messages would help you resolve it or post the content here.
Also please provide information about your system - which release you have? And if you have installed it from community plugins - was there any error message during installation? Check `/var/log/messages` logs in the host or in the HA jail.

**Comment by tprelog** [comment]
It's not looking good @adorobis -- Looks like they are blocking it from running

```
Home Assistant only supports Linux, OSX and Windows using WSL
```

**Reply by QuentinIII** [reply]
I can't send you the log this night, I'll send it tomorrow. 

I did see in some logs, this same message "Only work on Linux Windows WSL ..." With the same error.

I was able to start the configurator but not the core.

Also thanks for your reactivity, from a student french guy, this look really professionnal !

**Comment by tyme2tb** [comment]
Looks like the same issue as #52

**Reply by QuentinIII** [reply]
Yup the same problem, I have a friend who hass is working on truenas, he is working with truenas 12.0U7 with a hass on 12.2 P12

And for myself I'm working on truenas 12.0U8 and hass 12.2 P10

**Comment by tprelog** [comment]
Testing a workaround now

**Reply by QuentinIII** [reply]
Allright keep me updated ! By the way, I installed hass today but I'm having a 12.2 p10 build has my friend installed it a long time ago and has the 12.2 p12 build ? Does it matter ?

**Reply by tprelog** [reply]
> I installed hass today but I'm having a 12.2 p10 build has my friend installed it a long time ago and has the 12.2 p12 build ? Does it matter ?

No, not in this case

**Comment by tprelog** [comment]
I'll push an update to the plugin shortly... Fortunately it's a pretty simple fix you can apply yourself, instead of waiting for the update

instructions here https://github.com/tprelog/iocage-homeassistant/issues/52#issuecomment-1028397177

**Comment by tprelog** [comment]
Plugin is updated now... Since your just getting started @QuentinIII, I'd just destroy the existing plugin and do another install. It will be quicker than running a Plugin UPDATE

**Reply by QuentinIII** [reply]
Ok thanks, as you say I didn't try to upgrade it from shell just destroy the package ! I'll keep you up to date.
